Cargo.toml

```diff
@@ -17,13 +17,13 @@ path = "lib.rs"
 [features]
 default = []
 kv_unstable = ["log/kv_unstable_std", "slog/dynamic-keys"]
-record_location_unstable = []
+record_location_unstable = ["cached"]
 
 [dependencies]
 slog = "2.4"
 slog-scope = "4"
 log = { version = "0.4.11", features = ["std"] }
-cached = "0.23.0"
+cached = { version = "0.23.0", optional = true }
```

Comment:
BTW. I skimmed through `cached` and I don't understand why do you use it. Doesn't seem made for interning strings or anything. Can you explain what's the reasoning behind it?

Reply:
I've never done string interning in particular so I was looking for an ergonomic caching library which I thought would accomplish the same result.
I can do more digging on `string_cache` or `string_interner` crates if you'd prefer those, just let me know.

Reply:
I'm not in depth on interning, but I think they are good reasons why there are specialized libraries for them).